girder/setup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+requirements = [
+    'girder>=3.0.0a1'
+]
+
+setup(
+    author="Patrick Avery",
+    author_email='patrick.avery@kitware.com',
```

Comment:
These should point to Kitware as well

Reply:
Okay, I'll make the author `Kitware`. Should the email be `kitware@kitware.com`?

Reply:
Sure